### PR TITLE
6 extra languages

### PR DESCRIPTION
**Pull request by brandelune**
There is one string missing for Hindi, but I'll add it when I get it tomorrow. @goneall let me know if you see any other issue. This delivery should be way cleaner than the previous one.

### CODE REVIEW

**Review by goneall — APPROVED** [review]
LGTM - once the Hindi string is added, I'll merge these in and deploy to the web-app

**Comment by brandelune** [comment]
The Hindi fix is in, you can review now.

**Pull request merged**
